live/config-cdroot/fix_bootconfig

```diff
@@ -0,0 +1,95 @@
+#! /bin/bash
```

Comment:
This is incorrectly placed, I suppose that we need to have only arch specific versions here. I'll move it to the right directory.